### DISCUSSION TITLE
Just a little note: Use `or` instead of multiple `if` branches with a same result :)

### DISCUSSION THREAD

**Discussion by david-shiko** [python-telegram-bot/python-telegram-bot#2846] [Feature Suggestions]
https://github.com/python-telegram-bot/python-telegram-bot/blob/ade1529986f5b6d394a65372d6a27045a70725b2/telegram/ext/conversationhandler.py#L522

Example:
```
    if (not isinstance(update, Update)
        or update.channel_post
        or update.edited_channel_post
        or (self.per_chat and not update.effective_chat)
        or (self.per_message and not update.callback_query)
        or (update.callback_query and self.per_chat and not update.callback_query.message)):
        return None
```

Or `elif` also more "pythonic way"

**Comment by Bibo-Joshi** [comment]
It was like that before in fact  and was changed in #2131 :) IIRC it was mainly to make it more readable, but it could actually have something to do with one of our pre-commit hooks complaining after the settings where tightened.

`elif` would be unnecessary after a `return` …

In the end IMO it's also a bit a matter of taste 🙂
